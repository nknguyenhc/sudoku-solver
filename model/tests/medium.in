_ _ 6 1 _ 2 4 _ _
4 _ _ 5 _ _ _ _ 6
9 1 _ 7 _ _ _ 8 3
2 _ _ 9 _ 5 _ 7 _
_ _ _ _ _ _ _ _ 1
_ _ _ _ 8 _ 2 4 _
6 2 9 8 7 _ 3 _ _
_ _ 8 3 _ _ _ _ _
_ 7 _ 2 _ _ _ _ _