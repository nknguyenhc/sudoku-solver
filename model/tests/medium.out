5 8 6 1 3 2 4 9 7
4 3 7 5 9 8 1 2 6
9 1 2 7 6 4 5 8 3
2 4 3 9 1 5 6 7 8
8 6 5 4 2 7 9 3 1
7 9 1 6 8 3 2 4 5
6 2 9 8 7 1 3 5 4
1 5 8 3 4 9 7 6 2
3 7 4 2 5 6 8 1 9